SConstruct

```diff
@@ -713,20 +711,34 @@ conf.check_sysmacro_h()
 if conf.env['HAVE_LIBELF']:
     conf.env.Append(_LIBFLAGS=['-lelf'])
 
-# compiler optimisation and debug symbols:
-cc_O_option = '-O'
+if ARGUMENTS.get('GDB') == '1':
```

Comment:
``docs/developers.rst`` needs to be updated with the ``GDB`` and ``SYMBOLS`` variables.